src/services/misc.ts

```diff
@@ -0,0 +1,11 @@
+import request from '../utils/request';
```

Comment:
Codacy has a fix for the issue: [Replace `'../utils/request'` with `"../utils/request"`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
import request from "../utils/request";
```